src/app/components/button/button.module.scss

```diff
@@ -0,0 +1,130 @@
+.button {
+  @extend %transition-all;
+
+  cursor: pointer;
+
+  display: flex;
+  flex-flow: row nowrap;
+  gap: 8px;
+  place-content: center center;
+  align-items: center;
+
+  font-weight: $font-weight-medium;
+  letter-spacing: 0;
+  white-space: pre;
+
+  .arrow {
+    max-height: 24px;
+  }
+}
+
+.colored {
+  width: min-content;
+
+  &.large {
+    @include media-tablet {
+      width: 100%;
+    }
+
+    overflow: visible;
+
+    margin-top: 24px;
+    padding: 16px 24px;
+
+    font-family: $font-primary;
```

Comment:
You can omit specifying `font-family` explicitly, because `font-family` will be inherited from the nearest parent tag by default. In our case from the `Body` tag

https://www.codecademy.com/resources/docs/css/inheritance

![image](https://github.com/rolling-scopes/site/assets/114234698/bf89167a-f677-409d-9cff-e77abaa789bb)
![image](https://github.com/rolling-scopes/site/assets/114234698/0377263f-76f7-49da-bb69-a4a8c68465de)

Reply:
Removed